### ISSUE TITLE
Page rendered twice after redirect

### ISSUE THREAD

**Issue by newmen** [igor-alexandrov/wiselinks#11]
Hello,

I don't know this is a wiselinks bug or a problem of my application, but when there is a redirect the application processes redirected page twice.

```
Started GET "/postal_orders/new" for 127.0.0.1 at 2013-03-24 18:55:27 +0400
Processing by PostalOrdersController#new as HTML
Redirected to http://localhost:3000/delivery
Completed 302 Found in 2ms (ActiveRecord: 0.0ms)


Started GET "/delivery" for 127.0.0.1 at 2013-03-24 18:55:27 +0400
Processing by PagesController#show as HTML
  Parameters: {"identifier"=>"delivery"}
  Page Load (0.2ms)  SELECT "pages".* FROM "pages" WHERE "pages"."identifier" = 'delivery' LIMIT 1
  [wiselinks] Processing template request
  [wiselinks] title: Delivery – Beauty ribbons
Read fragment views/pages/2-20130322114251 (0.2ms)
  Rendered pages/_page.html.haml (1.6ms)
  Rendered pages/show.html.haml within layouts/wiselinks (2.4ms)
Completed 200 OK in 8ms (Views: 5.2ms | ActiveRecord: 0.2ms)


Started GET "/delivery" for 127.0.0.1 at 2013-03-24 18:55:27 +0400
Processing by PagesController#show as HTML
  Parameters: {"identifier"=>"delivery"}
  Page Load (0.2ms)  SELECT "pages".* FROM "pages" WHERE "pages"."identifier" = 'delivery' LIMIT 1
  [wiselinks] Processing template request
  [wiselinks] title: Delivery – Beauty ribbons
Read fragment views/pages/2-20130322114251 (0.2ms)
  Rendered pages/_page.html.haml (1.9ms)
  Rendered pages/show.html.haml within layouts/wiselinks (2.9ms)
Completed 200 OK in 9ms (Views: 5.7ms | ActiveRecord: 0.2ms)
```

However, in the javascript console I see only one request to draw a redirected page.

```
GET http://localhost:3000/postal_orders/new   302
GET http://localhost:3000/delivery   200
```

The problem on my side? :)

**Comment by igor-alexandrov** [comment]
Yes, problem is in wiselinks. I will try to fix it as soon as possible.

**Comment by igor-alexandrov** [comment]
Fixed in wiselinks-0.5.5.
Thanks for reporting.

P.S. You can check how redirection works on example application: http://wiselinks.herokuapp.com/

**Comment by newmen** [comment]
Thank you very much again!
